src/provider/provider_devdax_memory.c

```diff
@@ -369,9 +369,10 @@ static umf_result_t devdax_allocation_merge(void *provider, void *lowPtr,
 }
 
 typedef struct devdax_ipc_data_t {
-    char dd_path[PATH_MAX]; // path to the /dev/dax
-    size_t dd_size;         // size of the /dev/dax
-    size_t offset;          // offset of the data
+    char path[PATH_MAX]; // path to the /dev/dax
+    unsigned protection; // combination of OS-specific memory protection flags
+    size_t offset;       // offset of the data
```

Comment:
Could you please extend the comment regarding `offset` field?
As I understand it is offset from the base (e.g. beginning of the devdax mapping).